examples/oriented_images.js

```diff
@@ -0,0 +1,107 @@
+/* global itowns, document, renderer, initPicture  */
+// # Simple Globe viewer
+
+// Define initial camera position
+var positionOnGlobe = {
```

Comment:
This isn't really used since you're going to override it anyway.
I'd suggest to either:
- change `GlobeView` API and move `coordCarto` to `options` and if unset: don't modify the camera's position.
- use 0, 0, 0 in your example and add a comment to explain that's useless

I greatly prefer option 1: it should be easy to implement (but should go in its own PR)

Reply:
I understand, but I don't think it's a good idea to change GlobeView API.
Basically, I think it's a good idea to have a camera start position in the GlobeView, and it's usefull during dev process.
That being said, I've added a comment to explain that this variable is not very usefull and camera position will be overwritten to be placed in front of the picture later.